scripts/make_release_tarball.sh

```diff
@@ -7,7 +7,7 @@
 ##############################################################################
 
 TAR_CMD=gtar
-VERSION=1.0.0
+VERSION=1.0.1
```

Comment:
Where is our "magic" version symbol maintained?  I presume it does NOT need to be updated since this is simply a patch-level release.  Is that correct?

Reply:
Not merged yet.

Reply:
Oh right.